crash on eclipse#generateFiles

```
Caused by: java.lang.NullPointerException
	at org.jerkar.tool.builtins.eclipse.DotClasspathGenerator.writeModuleEntry(DotClasspathGenerator.java:299)
	at org.jerkar.tool.builtins.eclipse.DotClasspathGenerator.writeDependenciesEntries(DotClasspathGenerator.java:281)
	at org.jerkar.tool.builtins.eclipse.DotClasspathGenerator.generateJava(DotClasspathGenerator.java:248)
	at org.jerkar.tool.builtins.eclipse.DotClasspathGenerator._generate(DotClasspathGenerator.java:109)
	at org.jerkar.tool.builtins.eclipse.DotClasspathGenerator.generate(DotClasspathGenerator.java:93)
	at org.jerkar.tool.builtins.eclipse.JkBuildPluginEclipse.generateFiles(JkBuildPluginEclipse.java:60)
	... 12 more
```
Took a quick look at the source, looks like there's a missing null check on `jkAttachedArtifacts`?

I think I can fix it, I'll work on a PR...

--- a/org.jerkar.core/src/main/java/org/jerkar/tool/builtins/eclipse/DotClasspathGenerator.java
+++ b/org.jerkar.core/src/main/java/org/jerkar/tool/builtins/eclipse/DotClasspathGenerator.java
@@ -296,19 +296,17 @@ final class DotClasspathGenerator {
 
     private void writeModuleEntry(JkModuleId moduleId, XMLStreamWriter writer, JkResolveResult resolveResult,
             JkAttachedArtifacts jkAttachedArtifacts, Set<String> paths) throws XMLStreamException {
-        final Set<JkModuleDepFile> sourcesArtifacts = jkAttachedArtifacts.getArtifacts(moduleId, JkJavaBuild.SOURCES);
-        final File source;
-        if (!sourcesArtifacts.isEmpty()) {
-            source = sourcesArtifacts.iterator().next().localFile();
-        } else {
-            source = null;
-        }
-        final Set<JkModuleDepFile> javadocArtifacts = jkAttachedArtifacts.getArtifacts(moduleId, JkJavaBuild.JAVADOC);
-        final File javadoc;
-        if (!javadocArtifacts.isEmpty() && includeJavadoc) {
-            javadoc = javadocArtifacts.iterator().next().localFile();
-        } else {
-            javadoc = null;
+        File source = null;
+        File javadoc = null;
+        if (jkAttachedArtifacts != null) {
+            final Set<JkModuleDepFile> sourcesArtifacts = jkAttachedArtifacts.getArtifacts(moduleId, JkJavaBuild.SOURCES);
+            if (!sourcesArtifacts.isEmpty()) {
+                source = sourcesArtifacts.iterator().next().localFile();
+            }
+            final Set<JkModuleDepFile> javadocArtifacts = jkAttachedArtifacts.getArtifacts(moduleId, JkJavaBuild.JAVADOC);
+            if (!javadocArtifacts.isEmpty() && includeJavadoc) {
+                javadoc = javadocArtifacts.iterator().next().localFile();
+            }
         }
         writeClasspathEntry(writer, resolveResult.filesOf(moduleId).get(0), source, javadoc, paths);
     }

